src/js/view/controls/controlbar.js

```diff
@@ -305,6 +306,15 @@ export default class Controlbar {
         _model.change('fullscreen', (model, val) => {
             const fullscreenElements = [this.elements.fullscreen.element(), this.elements.imaFullscreen.element()];
 
+            if (fullscreenElements[0].getAttribute('aria-label') === 'Exit Fullscreen') {
+                // Uses setTimeout to prevent 'Exit Fullscreen Successful' voice line from being cutoff by other voice lines when using a screen reader
+                setTimeout(() => {
+                    this._exitFullscreenAnnouncer.textContent = 'Exit Fullscreen Successful';
+                }, 1450);
```

Comment:
Do we only need this because we use `aria-live=assertive`? If we were to switch to `aria-live=polite` would that solve the issue of needing a setTimeout? To me this setTimeout is going a bit into the magic number category. Not saying that isn't the right solution, but we will want to make sure that we try other options first. If we do keep the setTimeout around:
1. We need to make sure that we store it on `this` and clear it if we get back into this function faster than the timeout can run. 
2. We will also have to call clearTimeout on it in the `destroy` method, so that it doesn't happen after this object is destroyed and cause issues.

Reply:
I tried setting `aria-live=polite` but the voice line "Exit Fullscreen Successful" gets cut off by another voice line before it finishes. The `setTimeout` is very much a magic number as that is the only way I could get a clear voice line without it getting cut off. I will try and refactor to store it on `this` and add a `clearTimeout` in the `destroy` method.